deploy/cephcsi/image/Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN dnf install libcephfs-devel librados-devel librbd-devel /usr/bin/cc make -y
 
 ENV GOROOT=${GOROOT} \
     GOPATH=/go \
+    GO111MODULE=on \
```

Comment:
GOMODULE is on by default in go 1.13.x we don't need to set this explicitly

Reply:
What is the reason then that building fails? `go env` does not return `on` or `auto` it seems? Other variables that are enabled by default return a non-empty value with `go env`.

Maybe I am missing something?

Reply:
ok, I see it is `on` by default, but `go env GO111MODULE` returns `on` in that case too. By setting the variable empty on the cmdline `make GO111MODULE=""` the script detects it as `on`. By setting it to `off` or an other value, the script reports the error.

Reply:
`""` is also a valid value. we need to consider it

Reply:
Depending on the Golang version, `go env GO111MODULE` may return an empty string... Which means `on` or `auto`, so an empty-non-error-return should be valid too.